### DISCUSSION TITLE
Specific API endpoint does not work properly

### DISCUSSION THREAD

**Discussion by SarahJoyceMorgen** [neuefische/questions#806] [Web] [answered]
Hello, we have a question. It seems that we are encountering issues with the Details Page. Everything loads locally, but when trying to access the app with the provided link, it only shows a loading message. We have received feedback suggesting that the problem is likely related to the API. The API endpoint appears to be not working:

Vercel:
https://upcycling-e27gc46r6-kevinldg-projects.vercel.app

Pull Request:
https://github.com/kevinldg/upcycling-app/pull/26

https://github.com/kevinldg/upcycling-app

**Comment by kevinldg** [comment, marked as answer]
We've fixed the error. The problem was due to faulty returns. Thx @moonwave99 ! ❤️
